### PR TITLE
feat(i18n): add ui localization

### PR DESCRIPTION
**Pull request by ReenigneArcher**
## Description

Add localization to UI.

I followed this guide: https://vue-community.org/vue/translating-vue-js-applications-with-vue-i18n.html

Some formatting was removed from strings, especially in config descriptions. This was done to make it both easier to translate, and easier to contribute to the source code.

Todo:

- [x] Add remaining strings to `en.json`
- [x] Determine if translations should be in `/src_assets/common/assets/web/locale/` or `/locale/` (2nd option is my preference... but build works as is)
- [x] Determine if we should be using `/src_assets/common/assets/web/locale/sunshine.json` as the default file or `/src_assets/common/assets/web/locale/en/sunshine.json` (https://github.com/LizardByte/Themerr-plex/pull/398 will be a good test of how crowdin handles this)
- [x] Determine if the locale should be a config option or language switcher only available in the UI (decided on config option, unless there is an objection)
- [x] Ensure localization process is documented
- [x] Ensure build works
- [x] How to change the language with the config value?
- [x] html tags inside of strings dont' work. e.g. `<b>SOMETHING</b>`
  image
- [x] Lazy load languages https://kazupon.github.io/vue-i18n/guide/lazy-loading.html

### Issues Fixed or Closed

- Resolves https://ideas.moonlight-stream.org/posts/353/proposal-to-implement-localization-feature-in-sunshine

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency update (updates to dependencies)
- [x] Documentation update (changes to documentation)
- [ ] Repository update (changes to repository files, e.g. `.github/...`)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the in code docstring/documentation-blocks for new or existing methods/components

## Branch Updates
LizardByte requires that branches be up-to-date before merging. This means that after any PR is merged, this branch
must be updated before it can be merged. You must also
[Allow edits from maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I want maintainers to keep my branch updated

### CODE REVIEW

File: src_assets/common/assets/web/config.html [hunk 1]
```diff
@@ -37,6 +37,26 @@ <h1 class="my-4">Configuration</h1>
...
+            <option value="de">German</option>
+            <option value="en">English</option>
+            <option value="en_GB">English (UK)</option>
+            <option value="en_US">English (US)</option>
+            <option value="es">Spanish</option>
+            <option value="fr">French</option>
+            <option value="it">Italian</option>
+            <option value="ru">Russian</option>
+            <option value="sv">Swedish</option>
+            <option value="ch">Chinese Simplified</option>
```
Comment (ReenigneArcher) [inline src_assets/common/assets/web/config.html:53]: Todo: Use the languages spelling

File: src_assets/common/assets/web/Locale.vue [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import es from './locale/es.json'
+import fr from './locale/fr.json'
+import it from './locale/it.json'
+import ru from './locale/ru.json'
+import sv from './locale/sv.json'
+import zh from './locale/zh.json'
+
+// Create the i18n instance
+const i18n = createI18n({
+    locale: "en",  // initial locale, todo: how to get this from config?
```
Comment (ReenigneArcher) [inline src_assets/common/assets/web/Locale.vue:21]: Todo: How to get the locale from the config settings?

File: src_assets/common/assets/web/Locale.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    },
+});
+
+export {
+    i18n
+};
+
+export default {
+    created() {
+        this.fetchLocale();
```
Comment (ReenigneArcher) [inline src_assets/common/assets/web/Locale.vue:43]: Todo: determine why this isn't being called

Comment (TheElixZammuto) [reply src_assets/common/assets/web/Locale.vue]: Fixed in commit [b4c0a05](https://github.com/LizardByte/Sunshine/pull/2279/commits/b4c0a05b5a77ac8f484647663d239bedb035f246)

**Comment by ReenigneArcher** [comment]
Todo: placeholders are not working

**Comment by ReenigneArcher** [comment]
@cgutman FYI

**Pull request merged**